### ISSUE TITLE
canvas build failing

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#525]
Canvas is failing to build currently. I'll attach the npn-debug.log too. Console
 output during build:
````
make: Entering directory '/var/www/canvas/node_modules/karma/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/bufferutil/build'
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
In file included from ../src/bufferutil.cc:16:0:
../node_modules/nan/nan.h:328:47: error: 'REPLACE_INVALID_UTF8' is not a member of 'v8::String'
   static const unsigned kReplaceInvalidUtf8 = v8::String::REPLACE_INVALID_UTF8;
                                               ^
bufferutil.target.mk:86: recipe for target 'Release/obj.target/bufferutil/src/bufferutil.o' failed
make: *** [Release/obj.target/bufferutil/src/bufferutil.o] Error 1
make: Leaving directory '/var/www/canvas/node_modules/karma/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/bufferutil/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
     at ChildProcess.onExit (/usr/share/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:809:12)
gyp ERR! System Linux 3.16.0-4-amd64
gyp ERR! command "nodejs" "/usr/bin/node-gyp" "rebuild"
gyp ERR! cwd /var/www/canvas/node_modules/karma/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/bufferutil
gyp ERR! node -v v0.10.29
gyp ERR! node-gyp -v v0.12.2
gyp ERR! not ok
npm WARN This failure might be due to the use of legacy binary "node"
npm WARN For further explanations, please read
/usr/share/doc/nodejs/README.Debian

npm WARN optional dep failed, continuing bufferutil@1.2.1
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /var/www/canvas/npm-debug.log
npm ERR! not ok code 0
/usr/share/fab/product.mk:476: recipe for target 'build/stamps/root.patched' failed
make: *** [build/stamps/root.patched] Error 1
````

**Comment by JedMeister** [comment]
[npm-debug.log.txt](https://github.com/turnkeylinux/tracker/files/63396/npm-debug.log.txt)

**Comment by JedMeister** [comment]
Repegging to v14.2 for now...

**Comment by qrntz** [comment]
This should be closed now that https://github.com/turnkeylinux-apps/canvas/pull/10 is merged.

**Comment by JedMeister** [comment]
Oops I forgot to close this after we re-released v14.1
